pkg/utils/utils.go

```diff
@@ -38,6 +39,8 @@ const (
 var InitialState sriovnetworkv1.SriovNetworkNodeState
 var ClusterType string
 
+var pfPhysPortNameRe = regexp.MustCompile(`p\d+`)
```

Comment:
This regexp matches all the following names:
`enp3s0f0np0 enp3s0f0np0_0 enp3s0f0np0_1 enp3s0f0np0_2`
Maybe put a `$` at the end of expression?   regexp.MustCompile(`p\d+$`)

Reply:
This regex is for `/sys/class/net/,netdev>/phys_port_name` not the interface name
For PFs it is `p/d+` e.g `cat /sys/class/net/enp3s0f0np0/phys_port_name`  `p0`
For VF representors it is `pf/d+vf/d+` e.g `cat /sys/class/net/enp3s0f0np0_2/phys_port_name`  `pf0vf2`

Reply:
ack+